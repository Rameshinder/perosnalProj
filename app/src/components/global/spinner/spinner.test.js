import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Spinner from './index'

jest.mock('@anz/icon', () => 'Icon')

describe('<Spinner />', () => {
  it('Spinner snapshot test', () => {
    const component = shallow(<Spinner />)
    const tree = shallowToJson(component)
    expect(tree).toMatchSnapshot()
  })

  it('Spinner should render without crashing', () => {
    shallow(<Spinner />)
  })
})
