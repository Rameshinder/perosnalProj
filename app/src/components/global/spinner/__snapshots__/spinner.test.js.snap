// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> Spinner snapshot test 1`] = `
<styled.div>
  <styled.svg
    height="72"
    width="72"
  >
    <circle
      className="path"
      cx="36"
      cy="36"
      fill="none"
      r="34"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
  </styled.svg>
  <Icon
    iconName="Action Menu"
  />
</styled.div>
`;
